src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services9_compiled.php

```diff
@@ -299,7 +299,7 @@ protected function getMethodCall1Service()
 
         $instance->setBar($this->get('foo'));
         $instance->setBar(NULL);
-        $instance->setBar(($this->get("foo")->foo() . (($this->hasParameter("foo")) ? ($this->getParameter("foo")) : ("default"))));
+        $instance->setBar(($this->get('foo')->foo() . (($this->hasParameter("foo")) ? ($this->getParameter("foo")) : ("default"))));
```

Comment:
That's a minor, yet important side effect of the way this block of code is generated.
Will be even more visible in 3.3 where this will be something completely different.